assets/src/scripts/store/index.js

```diff
@@ -104,6 +105,48 @@ export const Actions = {
             );
         };
     },
+    retrieveExtendedTimeseries(site, period) {
```

Comment:
Now that we have this one, it's starting to feel like we could merge retrieveTimeseries, retrieveCompareTimeseries, and retrieveExtendedTimeseries together somehow.

Reply:
Maybe not too much could be merged - it's just starting to feel complex.

Reply:
It's probably worth thinking about. Maybe there are some helper functions that could help.